examples/inner-tx-create-assets/assets/coordinator.py

```diff
@@ -0,0 +1,131 @@
+"""
+NOTE: This is demo contracts, doesn't have security measures. Please don't use it in product.
+
+This smart contract doing:
+
+1. Demo create application, asset and use new appl and asset id after we create them 
```

Comment:
```suggestion
1. Creates a demo application, an asset and uses new appl and asset id after we created them 
```
This is a bit confusing I am not sure what you meant here. Could you check it one more time?

Reply:
Changed, please check @sczembor